view/frontend/web/js/navigation-sort.js

```diff
@@ -73,6 +74,53 @@ define([
                 return $(a).data(type) - $(b).data(type);
             }).appendTo(list);
         },
+
+        _handleFilterSearch: function () {
+            var filterInput = this.element.find('.tw_filtersearch');
```

Comment:
I see same DOM selection queries executed multiple times.
for example `filterInput.parent('div')`, `items.find(filterElement + ':visible')` etc. etc.
It's less readable and less performant. Please assign to a variable so each unique selector is only executed once.

Reply:
Changes implemented